doc/core/default_http_proxy_mapper.md

```diff
@@ -65,3 +65,24 @@ addresses as the host between the range `10.10.0.0` to `10.10.0.255`.
 
 The lookup and subsequent usage of an HTTP proxy for a specific channel can also
 be disabled by setting the channel arg `GRPC_ARG_ENABLE_HTTP_PROXY` to 0.
+
+## Address Proxy
+
+**Case 2** in the proposal documents a partially protected environment, where
+access to certain addresses must go through a proxy. Name resolution
+of protected servers works normally, and the proxy allows the CONNECT request
+to use an IP address instead of a hostname.
+
+Only requests for certain hosts must go through the proxy. Requests to other
+servers work without the proxy. Custom logic is used to determine which hosts
+the proxy will be used for.
+
+To use the address proxy, following parameters need to be specified:
```

Comment:
Suggest changing this to explicitly say "both of the following parameters", so that it's clear that they're both required.

Reply:
Done